scss/donate.scss

```diff
@@ -47,8 +47,9 @@ $animation-length: 80s;
   }
 }
 
-.hero {
-  position: relative;
+.hero-donate {
+  background-color: white;
+  overflow: hidden;
   padding-bottom: 10%;
 }
```

Comment:
i'm seeing an overlap of the donate hero for widths > ~1030px
![Screen Shot 2021-08-24 at 8 45 24 AM](https://user-images.githubusercontent.com/62481939/130618652-f84797ae-7f08-487e-8e1e-7705b90ac93e.png)